site/en/docs/extensions/reference/printing/index.md

```diff
@@ -59,17 +59,17 @@ const defaultPrinter = printers.find((printer) => {
 console.log(`Default printer: ${defaultPrinter.name}.\n\t${defaultPrinter.description}`);
 ```
 
-### submitJob()
+### submitJob() {: #submitjob }
 
 The `submitJob()` method requires three things.
 
 * A `ticket` structure specifying which capabilities of the printer are to be used. If the user needs to select from available capabilities, you can retrieve them for a specific printer using `getPrinterInfo()`.
-* A `SubmitJobRequest` structure, which specifies the printer to use, and the file or date to print. This structure contains a reference to the `ticket` structure. 
+* A `SubmitJobRequest` structure, which specifies the printer to use, and the file or date to print. This structure contains a reference to the `ticket` structure.
```

Comment:
I turned on an option in VSCode that trims whitespace differences. Sorry about that.